op-e2e/system/fees/fees_test.go

```diff
@@ -145,6 +178,9 @@ func testFees(t *testing.T, cfg e2esys.SystemConfig) {
 	sequencerFeeVaultStartBalance, err := l2Seq.BalanceAt(context.Background(), predeploys.SequencerFeeVaultAddr, big.NewInt(rpc.EarliestBlockNumber.Int64()))
 	require.Nil(t, err)
 
+	operatorFeeVaultStartBalance, err := l2Seq.BalanceAt(context.Background(), predeploys.OperatorFeeVaultAddr, big.NewInt(rpc.EarliestBlockNumber.Int64()))
```

Comment:
The fee vault start balances could also benefit from a zero-assertion (I mean all, not only op fee).

And this test's readability could certainly also benefit form a `balanceAt` helper function 😅

Reply:
Done